pkg/config/config.go

```diff
@@ -310,9 +310,12 @@ func initConfig(config Config) {
 	config.BindEnvAndSetDefault("logs_config.dd_url_443", "agent-443-intake.logs.datadoghq.com")
 	config.BindEnvAndSetDefault("logs_config.stop_grace_period", 30)
 
-	// Tagger full cardinality mode
-	// Undocumented opt-in feature for now
-	config.BindEnvAndSetDefault("full_cardinality_tagging", false)
+	// The cardinality of tags to send for checks and dogstatsd respectively.
+	// Choices are: low, orchestrator, high.
+	// WARNING: sending container tags for dogstatsd metrics may create more metrics
+	// (one per container instead of one per host). Changing this setting may have billing implications.
+	config.BindEnvAndSetDefault("checks_tag_cardinality", "orchestrator")
```

Comment:
isn't this different from the current behavior? can we keep this defaulting to low?